package.json

```diff
@@ -42,13 +42,13 @@
   "dependencies": {
     "@skyscanner/eslint-config-skyscanner": "^13.1.0",
     "@skyscanner/stylelint-plugin-backpack": "^2.0.1",
-    "stylelint": "^14.3.0",
+    "stylelint": "^14.16.1",
```

Comment:
This was already the case in the lockfile, making it explicit here as stylelint-scss has a peer dep on at least ^14.5.1